lib/src/main/java/com/auth0/android/lock/utils/CustomField.java

```diff
@@ -56,22 +58,66 @@
         int TYPE_EMAIL = 3;
     }
 
+    /**
+     * Location to store the field into.
+     */
+    @IntDef({PROFILE_ROOT, USER_METADATA})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface Storage {
+        /**
+         * Store the field into the user profile root.
+         * The list of attributes that can be added to your root profile is here https://auth0.com/docs/api/management/v2#!/Users/patch_users_by_id.
+         */
+        int PROFILE_ROOT = 0;
```

Comment:
Shouldn't metaData be 0 as it's the original and default with root being the new one.

Reply:
No need to swap these, but I was using the wrong constant value on the existing constructor so thanks for that 👍

Reply:
Reference https://github.com/auth0/Lock.Android/pull/543/files#diff-2c5ac7c3a0fefe698d1137e6668dc1d1R100